packages/tempus/src/Tempus_StepperForwardEuler_impl.hpp

```diff
@@ -185,18 +185,24 @@ void StepperForwardEuler<Scalar>::describe(
   Teuchos::FancyOStream               &out,
   const Teuchos::EVerbosityLevel      verbLevel) const
 {
-  out << std::endl;
-  Stepper<Scalar>::describe(out, verbLevel);
-  StepperExplicit<Scalar>::describe(out, verbLevel);
-  out << "  stepperFEAppAction_                = "
-      << stepperFEAppAction_ << std::endl;
-  out << "----------------------------" << std::endl;
+  auto l_out = Teuchos::fancyOStream( out.getOStream() );
+  Teuchos::OSTab ostab(*l_out, 2, this->description());
+  l_out->setOutputToRootOnly(0);
+
+  *l_out << std::endl;
+  Stepper<Scalar>::describe(*l_out, verbLevel);
+  StepperExplicit<Scalar>::describe(*l_out, verbLevel);
+  *l_out << "  stepperFEAppAction_ = "
+         << stepperFEAppAction_ << std::endl
```

Comment:
Is this actually printing the AppAction?

Reply:
It is printing out the RCP.  We should go with what is in IntegratorBasic::describe().  Printing the describe, if it is valid, otherwise print the RCP.  Done.

Reply:
OK. I see that now.

Reply:
Ok, AppActions do not have a describe function, so leaving as is.  Maybe in another PR we can add the describe function.